docs/documentation/usage.md

```diff
@@ -148,3 +148,37 @@ to date list of available options.
 - List firefox releases numbers
 
         mozregression --list-releases
+
+## Unprivileged user namespaces
+
+AppArmor can be used to restrict the usage of that kernel feature, which is
```

Comment:
"that kernel feature" -> "this kernel feature" or "aforementioned kernel feature"